backend/app/controllers/spree/admin/resource_controller.rb

```diff
@@ -3,7 +3,11 @@ class Spree::Admin::ResourceController < Spree::Admin::BaseController
 
   helper_method :new_object_url, :edit_object_url, :object_url, :collection_url
   before_action :load_resource, except: :update_positions
-  before_action :set_currency, :ensure_current_store, only: [:new, :create]
+  before_action only: [:new, :create] do
```

Comment:
@MatthewKennedy rather than doing a block it would be better to do have the old notation of methods - later it's possible to change that behaviour with skip_before_action